src/phoebus_driver.cpp

```diff
@@ -1068,6 +1069,10 @@ void UserWorkBeforeOutput(MeshBlock *pmb, ParameterInput *pin) {
         Real gam1[3][3];
         Real gam2[3][3];
         Real gam3[3][3];
+        const Real z = coords.Xc<3>(k, j, i);
+        const Real sigma = 2 * coords.CellWidthFA(X3DIR, k, j, i);
```

Comment:
this depends on the mesh width on the block you're on which makes me a bit uncomfortable. I suggest making sigma a free parameter that is set-able in the input deck.

Reply:
hmm that's a weird thing to have in input deck... plus out of context from everything else, no?

Reply:
done